examples/4-exchanges/src/index.re

```diff
@@ -1,13 +1,10 @@
 open ReasonUrql;
-open Client;
 
 /* This is the native debugExchange that ships with `urql`, re-implemented in Reason.
      Typically, you'd just add Exchanges.debugExchange to the Client's exchange array.
    */
-let debugExchange = exchangeInput => {
-  let forward = Exchanges.forwardGet(exchangeInput);
-
-  ops =>
+let debugExchange = (Exchanges.{forward}) =>
+  (. ops) =>
```

Comment:
The signature for exchanges had to change a bit. It was a happy accident that this was working before, made possible by the use of `forwardGet`, which forced compilation into a higher order function. But we really need to use uncurrying here to ensure the exchange actually returns a function that accepts operations and passes them through.